api/controllers/v0.3/DependentAccountAPI.js

```diff
@@ -170,7 +205,41 @@ const DependentAccountAPI = (args) => {
    * the parent patron data is already stored in memory but will be
    * overidden the next time a new parent patron's data is requested.
    */
-  const getAlreadyFetchedParentPatron = () => parentPatronData;
+  const getAlreadyFetchedParentPatron = () => {
+    if (!parentPatronData) {
+      return;
+    }
+
+    const varFields = parentPatronData.varFields || [];
+    let dependents;
+    const xFieldTags = varFields.filter((obj) => obj.fieldTag === "x");
+    // No varFields were found, so we can assume the patron doesn't
+    // have any dependent accounts yet.
+    if (xFieldTags.length === 0) {
+      dependents = "DEPENDENTS ";
```

Comment:
Is the space at the end of the string supposed to be there?

Reply:
It is. This is only for the response back to the requester so it's easy to add the newly created dependent juvenile barcode to this string. It's in `/api/controllers/v0.3/endpoints.js` L682 - https://github.com/NYPL/dgx-patron-creator-service/pull/58/files#diff-04f337212864a7293b4938d02be88954R682